@navikt/core/react/src/collapsible/Collapsible.types.ts

```diff
@@ -0,0 +1,23 @@
+export interface CollapsibleBaseProps {
+  /**
+   * The open state of the collapsible when it is initially rendered. Use when you do not need to control its open state.
+   */
+  defaultOpen?: boolean;
+  /**
+   * The controlled open state of the collapsible. Must be used in conjunction with onOpenChange.
+   */
+  open?: boolean;
+  /**
+   * Event handler called when the open state of the collapsible changes.
+   */
+  onOpenChange?: (open: boolean) => void;
+  /**
+   * When true, prevents the user from interacting with the collapsible.
```

Comment:
Høres ut som både content og trigger blir deaktivert, men gjelder bare trigger vel?
```suggestion
   * When true, prevents the user from interacting with the trigger.
```